pypowerwall/__init__.py

```diff
@@ -295,6 +295,10 @@ def poll(self, api='/api/site_info/site_name', jsonformat=False, raw=False, recu
             except:
                 log.debug('ERROR Unknown error connecting to Powerwall at %s' % url)
                 return None
+            if r.status_code == 404:
+                # API not found or no longer supported
+                log.debug('ERROR Powerwall API not found at %s' % url)
+                return None
```

Comment:
A 404 status response for an API call will now result in a `None` response and log a debug message.